src/app/countries.tsx

```diff
@@ -1,9 +1,11 @@
-import axios from "axios";
+import axios from 'axios';
+import type { CountryResponse } from '../models/CountryResponse';
+import type { Country } from '../models/Country';
 
-async function getCountries() {
+const getCountries = async () => {
```

Comment:
We can add a typing for the function return. Ex:
`const getCountries = async ():  Country[] => {`

Reply:
Well, I declared the typing inside of the function, other wise, I added your recommendation and I've get this warning
`Type '[Country]' is not a valid async function return type in ES5 because it does not refer to a Promise-compatible constructor value.` 🤔